core/src/com/unciv/ui/cityscreen/CityScreen.kt

```diff
@@ -186,6 +186,10 @@ class CityScreen(
         }
     }
 
+    fun canCityBeChanged(): Boolean {
+        return UncivGame.Current.worldScreen.canChangeState && !city.isPuppet
```

Comment:
We already reference this and have a local variable of "canChangeState", there shouldn't be a reason to reference the worldscreen twice

Reply:
FYI: (BaseScreen L23) UncivGame.Current is available as `game`